src/server/rollup.config.js

```diff
@@ -1,10 +1,12 @@
+import { cwd } from 'node:process';
 import { rollupPluginHTML as html } from '@web/rollup-plugin-html';
 import { join } from 'path';
 import { nodeResolve } from '@rollup/plugin-node-resolve';
 import { PATHS } from './visual-diff-plugin.js';
+import { cwd } from 'node:process';
```

Comment:
Copy-paste fail?

Reply:
Yes lol, this is what happens when I try to do something quickly in the GitHub UI 🤦